corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -422,6 +452,11 @@ def save(self, *args, **kwargs):
                 details={'pk': self.pk}
             )
 
+    class Meta:
+        index_together = (
+            ('user_id', 'device_id', 'app_id')
```

Comment:
In practice, I'm not sure how necessary this index is. Most users will have 1 app on one device, meaning that the current index on user_id will result in roughly the same number of disk I/O (perhaps even less when you consider that user_id index will be smaller

Reply:
Just to clarify, you're suggesting dropping this index entirely and only having the `user_id` index. And your reasoning is that since device_id and app_id are mostly unique per user that the index won't give a lot of value.

I think that makes sense. I'll update it.